backend/uclapi/common/decorators.py

```diff
@@ -198,21 +197,9 @@ def _check_temp_token_issues(token_code, personal_data, request_path, page_token
         })
         response.status_code = 400
         return response
-
-    # Check if TemporaryToken is still valid
-    existed = datetime.datetime.now() - temp_token.created
-
-    if existed.seconds > 300:
-        temp_token.delete()  # Delete expired token
-        response = JsonResponse({
-            "ok": False,
-            "error": "Temporary token expired."
-        })
-        response.status_code = 400
-        return response
-
+    print("No problem found with temp token")
```

Comment:
Do we still need this print statement for debugging?

Reply:
lul you wrote that :P